exosphere-shared/src/logger.spec.ls

```diff
@@ -0,0 +1,54 @@
+require! {
+  'chai': {expect}
+  'chalk' : {yellow}
+  './logger': Logger
+}
+
+describe 'Logger', ->
+
+  describe '_parse-line', ->
+    logger = new Logger [role]
+    role = 'exo-run'
+
+    specify 'should parse non-service log message correctly' ->
+      line = 'Attaching to exocom0.21.8, web'
+      logger._parse-line role, line, (left, right) ->
+        expect(left).to.eql(role)
+        expect(right).to.eql(line)
+
+    specify 'should parse service log message correctly' ->
+      line = '\u001b[33mweb             |\u001b[0m web server running at port 4000'
+      logger._parse-line role, line, (left, right) ->
+        expect(left).to.eql('\u001b[33mweb')
+        expect(right).to.eql('\u001b[0mweb server running at port 4000')
```

Comment:
Thoughts on having `_parse-line` completely strip out the colors? We can add back in our own colors after parsing.